storage/src/main/java/org/apache/kafka/storage/internals/log/UnifiedLog.java

```diff
@@ -1605,6 +1608,10 @@ private MemoryRecords trimInvalidBytes(MemoryRecords records, LogAppendInfo info
             // trim invalid bytes
             ByteBuffer validByteBuffer = records.buffer().duplicate();
             validByteBuffer.limit(validBytes);
+            if (isLeader) {
+                logger.warn("Trimming invalid bytes from message set for partition {}. Original size: {} bytes, valid bytes: {}, trimmed bytes: {}. ",
```

Comment:
We have the following log ident. So, there is not need to include partition explicitly.

`this.logIdent = "[UnifiedLog partition=" + topicPartition() + ", dir=" + parentDir() + "] ";`